ui-frontend/packages/catalog-ui-search/src/main/webapp/component/golden-layout/cross-window-communication.tsx

```diff
@@ -352,6 +355,44 @@ const useProvideWreqrEvents = ({
     (event: string, args: any, { doNotPropagate = false } = {}) => {
       if (goldenLayout && isInitialized) {
         if (event.startsWith('search') && !doNotPropagate) {
+          // Launch the 2D Map (openlayers) if it's not already open
```

Comment:
Should we avoid opening the map if the 3D map is already open?
Should we check if the 2D map is open in another window?
If the 2D map is minimized, would it be easy to maximize it?

Reply:
- Added a check for cesium as well as openlayers so it will now only launch openlayers if neither map is open.
- If the map is minimized the drawing tools are correctly added to that map, but there isn't an easy way to "maximize" that ContentItem, as currently when you want to maximize a visual you drag it to a position, we would be guessing what position and potentially disrupt the user's layout.
- Also will now find any map visuals open in a separate window.